### ISSUE TITLE
Typescript is off in Solid 

### ISSUE THREAD

**Issue by edygar** [sannajammeh/tw-classed#129]
**Describe the bug**
The simplest case causes TypeScript error.

**To Reproduce**
```typescript
import { classed } from "@tw-classed/solid";

const Wrapper = classed.div(
  "grid flex-1 items-center justify-center bg-red-200",
);

<Wrapper />
```

**Expected behavior**
No warning

**Screenshots**
<img width="935" alt="Screenshot 2023-11-29 at 09 33 29" src="https://github.com/sannajammeh/tw-classed/assets/566280/c0fd4651-0ae5-47fc-8a55-32fcdf643ee7">

**Environment:**
 - OS: Mac OS 14.1.1
 - Node.js Version: 18.17.1
 - Version: 1.1.6

**Comment by sannajammeh** [comment]
Its still experimental and Solid's api has changed a bit. Ill work on it this week!

**Comment by sannajammeh** [comment]
Appears to have been a bundler issue. Fixed once v1.7.0 lands
